File: swagger/models/get_client_visits_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .visit import Visit

class GetClientVisitsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		visits
		Contains information about client visits.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Visits': ['visits', Visit],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'visits': 'Visits',
		}

	pagination_response = None
	visits = None

File: swagger/Models/GetClientVisitsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetClientVisitsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about client visits.
	 * 
	 * @var Visit[] 
	 */
	public $visits;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Visits' => ['visits', 'array', Visit::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'visits' => ['Visits', 'array', Visit::class],
			];
	}
}
